### DISCUSSION TITLE
Open issues/Feature requests

### DISCUSSION THREAD

**Discussion by Spince** [RedashCommunity/redash#1] [Ideas]
Should we port over some of the existing stuff or start from scratch here?

**Comment by hazmeister** [comment]
Start from scratch I reckon. Bring things across as they get attention or someone wants to fix something?

**Comment by justinclift** [comment]
Yeah, there's hundred of open PRs on the main Redash.  For the short term, we'll need to be very, very selective to keep things manageable. :smile:

**Comment by justinclift** [comment]
@junnplus has resubmitted his CI PR, and that's just been merged.  Which means (in theory) this new repo shouldn't have busted CI any more.  Hopefully. :pray:

**Comment by justinclift** [comment]
k, we should probably start updating the very outdated dependencies.  PySAML (as per the recent vulnerability) was already done a while ago by Arik, but there are still others that need doing.

Does anyone feel like picking up that task, and getting it done over the next few days?

Hmmm, there's both Python and JS dependencies.  So we might want to have that as two tasks, depending on who wants to do the work. :smile:

**Comment by justinclift** [comment]
Oh, someone forked over the [setup](https://github.com/RedashCommunity/setup) repo.

That's a good idea.  Haven't checked yet, but there at least used to be quite a few outstanding PR's on the main `setup` repo that looked useful but never got attended to.

That's probably a good first task for someone looking for stuff to do. :smile:

ie cherry-pick across the useful ones from the main `setup` repo, and merge them in
